tests/integration/test_wpk/test_wpk_agent.py

```diff
@@ -333,13 +349,14 @@ def test_wpk_agent(get_configuration, prepare_agent_version, download_wpk,
                    configure_environment, start_agent):
     metadata = get_configuration['metadata']
     expected = metadata['results']
+    wazuh_log_monitor = FileMonitor(tools.LOG_FILE_PATH)
```

Comment:
Why do we instantiate a `FileMonitor` here, not use it and then recreate it at line 385?

Reply:
Done 29484935443c9d15880a267f5abcd72f8779aa3a